packages/esm-patient-list-app/src/api/api-remote.ts

```diff
@@ -136,6 +136,16 @@ export async function addPatientToList(data: AddPatientData, ac = new AbortContr
   return postData(`${cohortUrl}/cohortmember`, data, ac);
 }
 
+export async function removePatientFromList(cohortMembershipUuid: string, ac = new AbortController()) {
+  return deleteData(
+    `${cohortUrl}/cohortmember/${cohortMembershipUuid}`,
+    {
+      voidReason: '',
```

Comment:
We're not voiding the cohortMember, just adding an endDate to the record.

Reply:
Call the `deleteData` instead of `postData` @jwnasambu .

Reply:
@vasharma05 Deleting isn't the correct thing in this instance. Removing a patient from a cohort makes them no longer a cohort member, so their membership expires. It doesn't void the membership (that's a different operation).

@jwnasambu POST is better than DELETE; however, we need to post something more like the full existing record with an endDate (and remove the voidReason property)

Reply:
Hi @ibacher and @dkayiwa !
I tried both the DELETE and POST methods, but POST request didn't make any change.
Do we need to use the DELETE or the POST method?
Thanks!

POST Request:
```
curl --location 'https://dev3.openmrs.org/openmrs/ws/rest/v1/cohortm/cohortmember/90fec1c6-8799-4c65-98ea-9053f1568457' \
--header 'Content-Type: application/json' \
--header 'Cookie: JSESSIONID=375DCF7704E139C04853E6277331448F' \
--data '{
    "endDate": "2023-08-07T12:11:12.558Z"
}'
```

Reply:
P.S. DELETE method is deleting the cohortmember as expected.

Reply:
@dkayiwa , any update here ☝️

Reply:
https://issues.openmrs.org/browse/COH-54